playbooks/testenv/tasks/pre-deploy.yml

```diff
@@ -67,14 +67,12 @@
   - name: interfaces.d/eth0.cfg with NAT for floating IP pool
     template: src=../templates/etc/network/interfaces.d/eth0-controllers.cfg
               dest=/etc/network/interfaces.d/eth0.cfg
+    notify: bounce eth0
 
+  handlers:
   - name: bounce eth0
     shell: ifdown eth0; ifup eth0
 
-  - name: interfaces.d/br-ex.cfg for floating IP pool
-    template: src=../templates/etc/network/interfaces.d/br-ex.cfg
-              dest=/etc/network/interfaces.d/br-ex.cfg
```

Comment:
This got removed, but I don't see it added back in anywhere, and a later task attempts to manipulate this file.